package.json

```diff
@@ -31,10 +31,18 @@
   "author": "Mikeal Rogers <mikeal.rogers@gmail.com> (https://www.mikealrogers.com/)",
   "license": "(Apache-2.0 AND MIT)",
   "exports": {
-    "import": "./index.js"
+    ".": {
+      "import": "./index.js"
+    },
+    "./murmur32.js": {
+      "import": "./murmur32.js"
+    },
+    "./murmur128.js": {
```

Comment:
Is the `.js` on the end intentional here?  I think in most places we've dropped that.

For example: `import { cid } from 'multiformats/cid'`, etc.

Reply:
> importing specic implementation e.g. '@multiformats/murmur3/murmur128'

This isn't possible with the `.js` on the end, it has to be `import { ... } from '@multiformats/murmur3/murmur128.js'`

Reply:
That is correct and yeah my comment is incorrect there. It was intentional because browsers reject imports that do not have `.js` at the end. So I thought this made more sense. That said we can drop those to be more inline with multiformats/cid

Reply:
> browsers reject imports that do not have .js at the end

So all our current code with extension-less exports are not working in browsers without bundlers? Has this just _not_ been done before, or why haven't we heard this issue raised before since we're doing it pretty much universally now?

Reply:
I think bundlers take care of it. Anyway this seems controversial so I'll just drop this, we can revisit along with other libs when it's more relevant.